Sprint-1/interpret/time-format.js

```diff
@@ -22,3 +22,15 @@ console.log(result);
 // e) What do you think the variable result represents? Can you think of a better name for this variable?
 
 // f) Try experimenting with different values of movieLength. Will this code work for all values of movieLength? Explain your answer
+
+// a) 6 varieble declaration
+
+// b) one function call - console.log()
+
+// c) it divides movielength by 60 which will separate the seconds which is not enough to be one minute
```

Comment:
can you try explain in terms of the term "remainder".

Reply:
we divided movielength by 60, every 60 will be a minute, the remainder is the the seconds not enough to be a minute.  for example 2:43:4  the 4 is teh remainder seconds